p2p/starknet/adapters.go

```diff
@@ -0,0 +1,20 @@
+package starknet
```

Comment:
can we move these to `adapter` folder? similar to what I have done [here](https://github.com/NethermindEth/juno/pull/1086/commits/f0b3dee8aeb3ecbe369e6e274082a805fe54a460)

Reply:
Yep, moved.